articles/nodejs-environment-variables/index.md

```diff
@@ -63,11 +63,11 @@ Create a file called `index.js` and add the following code to it.
 
 ```js
 const express = require("express");
+require("dotenv").config();
+
 const app = express();
 const port = process.env.PORT || 3000;
 
-require("dotenv").config();
-
 app.listen(port, () => {
```

Comment:
This is absolutely correct. Great job in catching this error @AmrSaber!

When ```const port = process.env.PORT || 3000;``` is called before ```require("dotenv").config()``` it creates an issue where the PORT environment variable hasn't been loaded yet. This change fixes this 👍